build.gradle

```diff
@@ -0,0 +1,70 @@
+group = 'org.sagebionetworks'
+ext.name = 'bridge-base'
+version = '0.1.0+' + (new Date()).format(
+        'yyyyMMddHHmm', TimeZone.getTimeZone("UTC"))
+
+/*** Java ***/
+
+apply plugin: 'java'
+
+sourceCompatibility = '1.7'
+targetCompatibility = '1.7'
+
+jar {
+    group = project.group
+    baseName = project.ext.name
+    version = project.version
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile 'com.google.guava:guava:' + guavaVer
+    compile 'joda-time:joda-time:' + jodaVer
+    compile 'commons-codec:commons-codec:' + commonsCodecVer
+    compile 'org.apache.shiro:shiro-core:' + shiroVer
+    compile 'org.bouncycastle:bcprov-jdk15on:' + bcVer
+    compile 'org.bouncycastle:bcpkix-jdk15on:' + bcVer
+    testCompile 'junit:junit:' + junitVer
+}
+
+/*** IDE ***/
+
+apply plugin: 'eclipse'
+apply plugin: 'idea'
+
+/*** Code Quality Tools ***/
+
+apply plugin: 'findbugs'
+tasks.withType(FindBugs) {
+    reports {
+        xml.enabled = false
+        html.enabled = true
+    }
+}
+
+apply plugin: 'jdepend'
+tasks.withType(JDepend) {
+    reports {
+        xml.enabled = false
+        text.enabled = true
+    }
+}
+
+apply plugin: 'pmd'
+tasks.withType(Pmd) {
+    reports {
+        xml.enabled = false
+        html.enabled = true
+    }
+}
+
+apply plugin: 'jacoco'
```

Comment:
Can we set up Jacoco to fail the build if coverage numbers fall below a certain reasonable threshold? This makes sure we don't get a bunch of untested code getting checked in (which is important since this is a shared package across multiple components). At the very least, it'll force people to think about testing.